### PR TITLE
add isomorphic support

### PR DESCRIPTION
**Pull request by alejandromaka**
works for both server and client

### CODE REVIEW

File: lib/FlexModalWrapper.js [hunk 1]
```diff
@@ -74,7 +75,15 @@ ModalWrapper.propTypes = {
   closePortal: React.PropTypes.func
 };
 
+ModalWrapper = withStyles(css)(ModalWrapper);
+
 class FlexModalWrapper extends React.Component {
+  getChildContext() {
+    const isNode = typeof window === 'undefined' && typeof document === 'undefined';
```
Comment (alejandromaka) [inline lib/FlexModalWrapper.js]: will fix

**Comment by alejandromaka** [comment]
fixed.. also had a bug in there where i assumed css was in an array, which it wasnt. fixed now though

**Comment by PeteJodo** [comment]
looks alright to me. Just an fyi, the approach web-mc is taking is building 2 files, the bundle js file and a styles css file. The down side to that is that users have to include the css as well as the js file but it gets rid of having to use context.

Have you tried testing your branch with dotcom or admin?

**Comment by alejandromaka** [comment]
oh ok dont merge quite yet, i tested .com but i feel like i should test some more.
and yeah while using context kind of sucks, we dont need to worry about it. only withStyles wrapper class uses it to lazy load the styles:
https://github.com/kriasoft/isomorphic-style-loader/blob/master/src/withStyles.js#L26

**Pull request closed without merging**